playwright/tests/tlm-grapher.spec.ts

```diff
@@ -145,23 +173,36 @@ test('shrinks and expands a graph height', async ({ page, utils }) => {
 test('shrinks and expands both width and height', async ({ page, utils }) => {
```

Comment:
We do a bunch of shrink and expand tests but I'd like to see you add a grid layout and prove we can put 4 in a grid pattern.